cmd/clusterctl/client/cluster/mover.go

```diff
@@ -1329,7 +1329,13 @@ func applyMutators(object client.Object, mutators ...ResourceMutatorFunc) (*unst
 	}
 	u.SetUnstructuredContent(to)
 	for _, mutator := range mutators {
-		if err := mutator(u); err != nil {
+		var err error
+		if mutator != nil {
```

Comment:
How about like this ?

```go
func applyMutators(object client.Object, mutators []ResourceMutatorFunc) (*unstructured.Unstructured, error) {
```

Originally, applyMutators receives variable length arguments. In addit this function is private method